server/src/test/java/org/spine3/server/projection/ProjectionRepositoryShould.java

```diff
@@ -496,31 +497,32 @@ private static TaskAdded taskAdded() {
     }
 
     /** Stub projection repository. */
-    private static class TestProjectionRepository extends ProjectionRepository<ProjectId, TestProjection, Project> {
-        protected TestProjectionRepository(BoundedContext boundedContext) {
+    private static class TestProjectionRepository
+            extends ProjectionRepository<ProjectId, TestProjection, Project> {
+        private TestProjectionRepository(BoundedContext boundedContext) {
             super(boundedContext);
         }
 
-        protected TestProjectionRepository(BoundedContext boundedContext, Duration catchUpMaxDuration) {
-            super(boundedContext, true, catchUpMaxDuration);
-        }
-
+        @SuppressWarnings("unused")
         @Subscribe
         public void apply(ProjectCreated event, EventContext eventContext) {
             // NOP
         }
     }
 
     /** Stub projection repository with the disabled automatic catch-up */
-    private static class ManualCatchupProjectionRepository extends ProjectionRepository<ProjectId, TestProjection, Project> {
+    private static class ManualCatchupProjectionRepository
+            extends ProjectionRepository<ProjectId, TestProjection, Project> {
         protected ManualCatchupProjectionRepository(BoundedContext boundedContext) {
             super(boundedContext, false);
         }
 
-        protected ManualCatchupProjectionRepository(BoundedContext boundedContext, Duration catchUpMaxDuration) {
+        protected ManualCatchupProjectionRepository(BoundedContext boundedContext,
+                                                    Duration catchUpMaxDuration) {
             super(boundedContext, false, catchUpMaxDuration);
         }
 
+        @SuppressWarnings("unused")
         @Subscribe
         public void apply(ProjectCreated event, EventContext eventContext) {
```

Comment:
Maybe we should finally add an IDEA config to avoid suppressing handler methods as `unused`?